content/en/docs/getting-started/deployment.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Deployment
+description: Installing Falco on a Cluster
+weight: 4
+---
+
+## Kubernetes
+
+Falco can be deployed in Kubernetes as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) to monitor the system events in each node of your cluster.
+
+### Helm
+
+One of the easiest ways to install Falco in Kubernetes is by using [Helm](https://v3.helm.sh/docs/intro/install/). The Falco community supports an official helm chart, and documentation on how to use it can [be found here](https://github.com/falcosecurity/charts/tree/master/falco). The chart [can be easily configured](https://github.com/falcosecurity/charts/tree/master/falco#configuration) to fit the most common installation requirements.
+
+### DaemonSet
+
+Falco can also be installed in Kubernetes manually. In this case, you are in charge of providing the DaemonSet object YAML definition and deploying it in your cluster. For further details, [you can find an example here](https://github.com/falcosecurity/evolution/tree/master/deploy/kubernetes/falco/templates).
+
+### Docker Deprecation in Kubernetes
+
+Since version 1.20, **Kubernetes started deprecating the Docker runtime** in favor of runtimes that use the Container Runtime Interface (CRI). You can find further details on [kubernetes.io](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/). In the versions of Kubernetes where Docker has been deprecated, Falco will not be able to obtain container information from the Docker socket anymore. 
+
+Starting from [version 0.15.0](https://github.com/falcosecurity/falco/blob/master/CHANGELOG.md#v0150), Falco supports CRI-compatible runtimes such as containerd or CRI-O. A custom container runtime socket path can be specified by using `--cri` command line argument of Falco, or by setting the `containerd.socket` parameter in the official Helm chart. By default, Falco tries to use the containerd socket at `/run/containerd/containerd.sock`. **If you run a recent version of Falco, no further action is required**.
+
+As many Falco use cases are out of the scope of Kubernetes, **Falco will keep supporting Docker as a container runtime**. Generally, Falco will keep looking for the Docker socket to obtain container information. If not available, it will automatically attempt using the other supported container runtimes, which include the CRI-compatible ones.
```

Comment:
so if dockerd is running on the machine, falco will try to connect to it by default, even if k8s is using an alternative runtime? This may be a problem on some environments. How users of these environments can detect this mismatch? Will the change of `--cri` parameter or setting of `containerd.socket` will fix the mismatch?

Maybe this section can be structured to reflect what actions needs to be taken in what scenarios? For example - if older version than XX is used - please update. If you see that Falco failed to detect CRI or detected incorrectly (how?) - set these settings, etc.

Reply:
oh, just read your comment in the doc https://docs.google.com/document/d/1ZFi4uKit63ga5sxEiZblfb-c23lFhvy6RXVPikS8wf0/edit#. It seems that if two runtimes detected, both will be used. This is cool. Still, more step by step instruction may be helpful. Plus if there is a way to detect that CRI wasn't connected, it will help

Reply:
Hi @SergeyKanzhelev! Thanks for the feedback, those are all good points. I edited the last commit trying to address your concerns and add more context in this section. Let me know what you think.

Reply:
Hey @SergeyKanzhelev 

I believe all your concerns should be addressed now. So I'm going to approve this PR.
Let us know if there's anything missing.

Thank you!

Reply:
sorry for delay. Updated version is great!